Dockerfile

```diff
@@ -25,17 +33,12 @@ RUN yarn run build-css
 
 # Build the production image
 # ===
-FROM ubuntu:jammy
+FROM base AS production
 
-# Install python and import python dependencies
-RUN apt-get update && apt-get install --no-install-recommends --yes python3 python3-setuptools ca-certificates libsodium-dev
-COPY --from=python-dependencies /root/.local/lib/python3.10/site-packages /root/.local/lib/python3.10/site-packages
-COPY --from=python-dependencies /root/.local/bin /root/.local/bin
 WORKDIR /srv
 COPY . .
-ENV PATH="/root/.local/bin:${PATH}"
-RUN rm -rf package.json yarn.lock .babelrc webpack.config.js requirements.txt
```

Comment:
Don't we need to remove these anymore from production image?

Reply:
I guess we are removing them for our other sites because they are not needed there. 

With your changes we are not removing them anymore. So they will end up in production image, or am I missing something/

Reply:
Yes, that the impact of this change, but if we follow that logic we should also remove scss files, konf files..

So either this command is not doing it's purpose well or maybe it's there of other security purposes? But not sure exactly.

Reply:
Personally, I feel this is unnecessary hence the change.

Reply:
Fair. Fine with me.